pom.xml

```diff
@@ -1,8 +1,8 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.axonivy.market</groupId>
-  <name>my-product</name><!-- fill your product name -->
-  <artifactId>my-product-modules</artifactId><!-- fill your product name + a "-modules" postfix -->
+  <name>jpa-persistence-lib</name>
```

Comment:
I think it would make sense to streamline this name with our repository name, which is persistence-utils.

https://github.com/axonivy-market/persistence-utils